projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.spec.ts

```diff
@@ -78,11 +69,15 @@ describe('ListViewComponent', () => {
         TestBed.configureTestingModule({
             declarations: [
                 ListViewComponent,
+                ResourceListComponent,
```

Comment:
You shouldn't use the actual components but mock them like here https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/9fb5841c9c62555032859c13819d5252ca4d4fac

The idea is that you provide a mocked component with the same selector and the same inputs and outputs as the real component.

Reply:
so I think in your case you have to mock the `ResourceListComponent` and the `ResourceGridComponent`.

The mocks can have an empty template.

Reply:
I see, this is what we had (but probably not correct). I will try to undo...

Reply:
Now I understand. Thanks for explanation.
Resolved in d4c83b0